### DISCUSSION TITLE
A way to utilize on mobile?

### DISCUSSION THREAD

**Discussion by simpL3gr** [Anarios/return-youtube-dislike#346] [Ideas]
The extension is usable on mobile browsers but not the youtube app. Maybe an idea would be to create a small android app for example that does the below: when watching a video, you can choose to share the url with that app which in turn checks the dislikes and shows them as a notification or "on screen bubble" or something. It can also give you a floating dislike button if you want it.

**Comment by hofq** [comment]
Yes, the YT Vanced Team Seems to be working on this [Vanced/1394](https://github.com/YTVanced/Vanced/issues/1394#issuecomment-992475003), so if you are comfortable using Vanced, there is a chance they integrate it

**Reply by severtheskyline** [reply]
It seems the extension and the user script works on the mobile website now, for not going all the way with extensions in Firefox, you can use the user script in Bromite.

**Comment by sy-b** [comment]
Vanced team has implemented this.
Enjoy watching on android 👍
